lib/improver/tests/acceptance/test_convert_to_realizations.py

```diff
@@ -28,24 +28,46 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
+"""
+Tests for the convert-to-realizations CLI
+"""
 
-. $IMPROVER_DIR/tests/lib/utils
+import pytest
 
-@test "convert-to-realizations --no-of-realizations 12 input --output output" {
-  improver_check_skip_acceptance
-  KGO="percentiles-to-realizations/percentiles_rebadging/kgo.nc"
+from improver.tests.acceptance import acceptance as acc
 
-  # Run Ensemble Copula Coupling to convert one set of percentiles to another
-  # set of percentiles, and then rebadge the percentiles to be ensemble
-  # realizations.
-  run improver convert-to-realizations --no-of-realizations 12 \
-      "$IMPROVER_ACC_TEST_DIR/percentiles-to-realizations/percentiles_rebadging/multiple_percentiles_wind_cube.nc" \
-      --output "$TEST_DIR/output.nc"
-  [[ "$status" -eq 0 ]]
+pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
+CLI = acc.cli_name_with_dashes(__file__)
+run_cli = acc.run_cli(CLI)
 
-  improver_check_recreate_kgo "output.nc" $KGO
 
-  # Run nccmp to compare the output and kgo.
-  improver_compare_output "$TEST_DIR/output.nc" \
-      "$IMPROVER_ACC_TEST_DIR/$KGO"
-}
+def test_percentiles(tmp_path):
```

Comment:
This test takes ~10 seconds to run - please mark it as slow with `@pytest.mark.slow`.